collections/file_systems/roles/xfs_quota/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+user_quotas_default_soft: 5G
```

Comment:
In BlueBanquise, roles variables must be prefixed by role name. It was introduced to avoid names collision between roles.
Can you update variables names according to this convention ? 🙂